src/atoms/forms/submit-button.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+// -----------------------------------------------------------------------------------------
+// #region Interfaces
+// -----------------------------------------------------------------------------------------
+
+export interface SubmitButtonProps {
+    buttonText?: string;
+    cssClassName?: string;
+
+    /**
+     * Typically you do not need this, but using it can allow you to
+     * place the button outside of the form for styling purposes.
+     */
+    formId?: string;
+}
+
+// #endregion Interfaces
+
+// -----------------------------------------------------------------------------------------
+// #region Component
+// -----------------------------------------------------------------------------------------
+
+const SubmitButton: React.FC<SubmitButtonProps> = (
+    props: SubmitButtonProps
+) => {
+    return (
+        <input
+            className={props.cssClassName || "c-button"}
+            form={props.formId}
+            type="submit"
+            value={props.buttonText || "Submit"}
```

Comment:
Nitpick - prefer `??` vs `||` for null coalescing. (Assuming this library is on the version of CRA/Typescript that supports the operator)